KarhooUISDK/Screens/CheckoutScreen/Components/AddPaymentView/KarhooAddPaymentView.swift

```diff
@@ -159,9 +162,11 @@ final public class KarhooAddPaymentView: UIView, AddPaymentView {
     }
     
     func set(nonce: Nonce) {
-        passengerPaymentTitle.text = UITexts.Payment.paymentMethod + " **** " + nonce.lastFour
+        passengerPaymentTitle.text = "\(UITexts.Payment.paymentMethod) **** \(nonce.lastFour)"
         passengerPaymentSubtitle.text = UITexts.Generic.edit
 		passengerPaymentImage.image = UIImage.uisdkImage(nonce.cardType)
+        accessibilityLabel = "\(UITexts.Payment.paymentMethod): **** \(nonce.lastFour.split(separator: " ").joined())"
```

Comment:
do we need the `****` in the accessibility label?

Reply:
We suppose to just read the content of label I guess